src/locale/properties/shared/shared.en.properties

```
@@ -144,3 +144,6 @@ shared.worksDescription=Add your research outputs such as publications, data set
 shared.worksLearnMore=Learn more about adding works to your ORCID record
 shared.employmentDescription=Add details of your current and previous employers.
 shared.employmentLearnMore=Learn more about adding employment information to your ORCID record
+share.addAnotherId=Add other identifier
```

@leomendoza123 There is a typo here 
```suggestion
shared.addAnotherId=Add other identifier
```